docker/Dockerfile

```diff
@@ -38,4 +38,6 @@ USER zipkin
 
 EXPOSE 9410 9411
 
+HEALTHCHECK --interval=10s --start-period=30s --timeout=5s CMD wget --quiet http://localhost:9411/health || exit 1
```

Comment:
Cool, I didn't realize we were using the debug image

Reply:
does this imply any blocking? the start period is really long

Reply:
Doesn't block, it probes every interval regardless - went ahead and lowered interval to 5s. Any failures during the start-period aren't counted as "unhealthy", they're just skipped, but they still happen.

Reply:
@devinsba I didn't notice the debug image until now also. In fact my hotel can't even pull the zipkin image it is so big. I opened this up for follow-up in case there's a way to reduce size again https://github.com/openzipkin/docker-zipkin/issues/226